Content.Server/StationEvents/BasicStationEventSchedulerSystem.cs

```diff
@@ -43,16 +42,21 @@ public override void Initialize()
 
             // Can't just check debug / release for a default given mappers need to use release mode
             // As such we'll always pause it by default.
-            _configurationManager.OnValueChanged(CCVars.EventsEnabled, value => RuleAdded = value, true);
+            _configurationManager.OnValueChanged(CCVars.EventsEnabled, SetEnabled, true);
```

Comment:
does onvaluechanged get run when cvars are initially set on the server?

Reply:
> does does this even subscribe soon enough to catch it initially being set, i have a feeling its always going to default to off now

Even if it doesn't subscribe in time, the last option to OnValueChanged, `invokeImmediately` ensures that it will automatically call the function once to set the current value. 

I'm not sure whether the on-change always gets invoked at least once, or only if the config/command-line value is not the cvar's default, but AFAIK this should all work fine, and the enabled definitely gets set to true when testing locally.